netdeploy/remote/deployedNetwork_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2019-2021 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package remote
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/protocol"
+)
+
+func Test_createSignedTx(t *testing.T) {
+	var networkState netState
+	networkState.nApplications = 2
+	networkState.nAssets = 2
+	networkState.nAccounts = 10
+	networkState.roundTrxCnt = 4
+	networkState.txState = protocol.PaymentTx
+
+	params := config.Consensus[protocol.ConsensusCurrentVersion]
+
+	secretDst := keypair()
+	src := basics.Address(secretDst.SignatureVerifier)
+
+	//create accounts
+	sntxs, _ := createSignedTx(src, basics.Round(1), params, &networkState)
+	require.Equal(t, 4, len(sntxs))
+	require.Equal(t, protocol.AssetConfigTx, networkState.txState)
+	for _, sntx := range sntxs {
+		require.Equal(t, protocol.PaymentTx, sntx.Txn.Type)
+	}
+
+	initialAccounts := networkState.accounts
+
+	//should be creating assets next
+	sntxs, _ = createSignedTx(src, basics.Round(1), params, &networkState)
+	accounts := networkState.accounts
+	require.Equal(t, 2, len(sntxs))
+	require.Equal(t, protocol.ApplicationCallTx, networkState.txState)
+	require.Equal(t, uint64(0), networkState.nAssets)
+	//same accounts should be used
+	require.Equal(t, initialAccounts[0], accounts[0])
+	for _, sntx := range sntxs {
+		require.Equal(t, protocol.AssetConfigTx, sntx.Txn.Type)
+	}
+
+	//should be creating applications next
+	sntxs, _ = createSignedTx(src, basics.Round(1), params, &networkState)
+	require.Equal(t, 2, len(sntxs))
+	require.Equal(t, protocol.PaymentTx, networkState.txState)
+	require.Equal(t, uint64(0), networkState.nApplications)
+	require.Equal(t, initialAccounts[0], accounts[0])
+	for _, sntx := range sntxs {
+		require.Equal(t, protocol.ApplicationCallTx, sntx.Txn.Type)
+	}
+
+	//	create payment transactions for the remainder rounds
```

Comment:
a bit more consistency in comments: this file has `//comment`, `//.    comment` , `// comment`. The latter IMO preferable.